src/Api/SecretsManager/Controllers/SecretsController.cs

```diff
@@ -74,7 +73,9 @@ public async Task<SecretResponseModel> CreateAsync([FromRoute] Guid organization
 
         var userId = _userService.GetProperUserId(User).Value;
         var result = await _createSecretCommand.CreateAsync(createRequest.ToSecret(organizationId), userId);
-        return new SecretResponseModel(result);
+
+        // Creating a secret means you have read & write permission.
+        return new SecretResponseModel(result, false, false);
```

Comment:
```suggestion
        // Creating a secret means you have read & write permission.
        return new SecretResponseModel(result, true, true);
```

Based on the comment, wouldn't having access be true, true?

Reply:
Yes, good catch.